mitol-django-digital-credentials/mitol/digitalcredentials/backend.py

```diff
@@ -0,0 +1,45 @@
+"""Digital credentials proxy to sign-and-verify service"""
+import json
+from copy import deepcopy
+from typing import Dict
+from urllib.parse import urljoin
+
+import requests
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+from django.utils.module_loading import import_string
+
+from mitol.common.utils import now_in_utc
+from mitol.digitalcredentials.models import LearnerDID
+
+
+def build_api_url(path: str) -> str:
+    """Build a url for the verify service"""
+    base_url = settings.MITOL_DIGITAL_CREDENTIALS_VERIFY_SERVICE_BASE_URL
+    if not base_url:
+        raise ImproperlyConfigured(
+            "MITOL_DIGITAL_CREDENTIALS_VERIFY_SERVICE_BASE_URL is not set"
+        )
+    return urljoin(base_url, path)
+
+
+def build_credential(courseware_object: Dict, learner_did: LearnerDID) -> Dict:
+    """Build the credential"""
+    build_credendial_func_name = (
+        settings.MITOL_DIGITAL_CREDENTIALS_BUILD_CREDENTIAL_FUNC
+    )
+    if not build_credendial_func_name:
+        raise ImproperlyConfigured(
+            "MITOL_DIGITAL_CREDENTIALS_BUILD_CREDENTIAL_FUNC is not set"
+        )
```

Comment:
I think that falls in a similar vein to feedback @gsidebo left on my PR for the mail app about environment variable parsing, my response to that is [here](https://github.com/mitodl/ol-django/pull/5#discussion_r488977547).

I think the answer is that however we end up doing that, the equivalent call to [`envs.validate()`](https://github.com/mitodl/mitxpro/blob/73bd05080adf2309b0147bb7e1cbf88caf08ac33/mitxpro/apps.py#L15) would handle this.

Definitely something to bite off sooner rather than later though,